src/Api/Models/Response/ProfileOrganizationResponseModel.cs

```diff
@@ -32,8 +32,12 @@ public ProfileOrganizationResponseModel(OrganizationUserOrganizationDetails orga
         UsePasswordManager = organization.UsePasswordManager;
         UsersGetPremium = organization.UsersGetPremium;
         UseCustomPermissions = organization.UseCustomPermissions;
-        UseActivateAutofillPolicy = organization.PlanType == PlanType.EnterpriseAnnually ||
-            organization.PlanType == PlanType.EnterpriseMonthly;
+        UseActivateAutofillPolicy = organization.PlanType is PlanType.EnterpriseAnnually2019 or
```

Comment:
⛏️ Could be worth setting up a `PlanType.IsEnterprise` extension method. Potentially along with `IsFamily` and `IsTeams` if we need them.

Reply:
Good suggestion! I took a second pass at this. Instead of writing a set of extension methods, I went ahead with using the existing [ProductType](https://github.com/bitwarden/server/blob/AC-1650-Increase-Enterprise-and-Teams-Plans-by-1/src/Core/Enums/ProductType.cs) enum on the [Plan](https://github.com/bitwarden/server/blob/AC-1650-Increase-Enterprise-and-Teams-Plans-by-1/src/Core/Models/StaticStore/Plan.cs#L8C15-L8C15) object, which is the same for all PlanTypes. Let me know what you think about it

Reply:
There's a good argument in my mind for consolidating `ProductType` and `PlanType`, too. Might be worth it to add this as tech debt